github.com.hashicorp.go.kms.wrapping.v2.types.proto

```diff
@@ -119,3 +133,33 @@ enum HmacType {
 	SHA384 				= 3;
 	SHA512 				= 4;
 }
+ 
+// KeyType defines the key's type
+  enum KeyType {
+	KEY_TYPE_UNKNOWN 	= 0;
+	RSA2048 			= 1;
+	RSA3072 			= 2;
+	RSA4096 			= 3;
+	AES256_GCM96 		= 4;
```

Comment:
Technically you're mixing keys and algorithms here. Some are key types, like RSA2048. But AES-GCM is an algorithm that uses an AES key, one of many. Probably this should just be AES256.

Reply:
ty!